pkg/cmd/server/origin_master.go

```diff
@@ -83,10 +85,18 @@ func (cfg Config) BuildOriginMasterConfig() (*origin.MasterConfig, error) {
 		return nil, err
 	}
 
+	dnsAddr := cfg.DNSBindAddr
```

Comment:
It's not appropriate for it to be called more than once, so I don't want to encourage reuse.


> On Mar 10, 2015, at 7:46 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/cmd/server/origin_master.go:
> 
> > @@ -83,10 +85,18 @@ func (cfg Config) BuildOriginMasterConfig() (*origin.MasterConfig, error) {
> >  		return nil, err
> >  	}
> >  
> > +	dnsAddr := cfg.DNSBindAddr
> This validation should happen in a separate method call for GetDNSAddr like we do for other addresses that require detection. https://github.com/openshift/origin/blob/master/pkg/cmd/server/config.go#L123 as an example.
> 
> —
> Reply to this email directly or view it on GitHub.
>